arch/RISCV/RISCVGenAsmWriter.inc

```diff
@@ -1187,7 +1187,7 @@ static void printInstruction(MCInst *MI, SStream *O, MCRegisterInfo *MRI)
   };
 
 #ifndef CAPSTONE_DIET
-  SStream_concat0(O, "\t");
+  //SStream_concat0(O, "\t");
```

Comment:
you can remove this altogether, rather than commenting it out.

Reply:
I will update TableGen patch later.

Reply:
https://travis-ci.org/aquynh/capstone/jobs/501114594
This error log looks like no reason at all.

Reply:
the compiler error? i think that is something, because compiler is rarely wrong. we have no choice, but to fix that for CI.

Reply:
how it possible `uint32_t` is `int`?

Reply:
In the source, the declaratrion is `uint32_t Insn`.

Reply:
> the compiler error? i think that is something, because compiler is rarely wrong. we have no choice, but to fix that for CI.

looks like all error clear.